projects/packages/forms/package.json

```diff
@@ -37,6 +37,7 @@
 		"@automattic/jetpack-analytics": "workspace:*",
 		"@automattic/jetpack-components": "workspace:*",
 		"@automattic/jetpack-shared-extension-utils": "workspace:*",
+		"@automattic/components": "2.2.0",
```

Comment:
We'll need v2.3.0 where the Badge actually will be:
- https://github.com/Automattic/wp-calypso/pull/103160